vanilla-framework/vanilla-components
# Add working Accordion pattern

## Done

Add working Accordion pattern

## QA

- Pull branch
- Run `npm run start`
- View pattern at: http://localhost:3000/accordion
- Run `npm run test` - verify no errors
- Run `npm run lint` - verify no errors

Fixes #27
